FEATURES.md

```diff
@@ -0,0 +1,17 @@
+# Proposing Features
+
+A feature request or proposal is an issue opened on a repo in opensearch-project that states an end-user problem, and proposes a solution in the form of a feature. It should meet the following criteria.
+
+- Is a feature.
```

Comment:
I think we're missing something about describing the feature.  Why do we want to add this?  What problem/use case is expected to solve?

Reply:
Changing to "Contains a real use-case or describes a problem."